apps/heft/src/schemas/api-extractor.schema.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "API Extractor Task Configuration",
+  "description": "Defines the heft configuration for the API Extractor task.",
+  "type": "object",
+
+  "additionalProperties": false,
+
+  "properties": {
+    "$schema": {
+      "description": "Part of the JSON Schema standard, this optional keyword declares the URL of the schema that the file conforms to. Editors may download the schema and use it to perform syntax highlighting.",
+      "type": "string"
+    },
+
+    "useProjectTypescriptVersion": {
+      "type": "boolean",
+      "description": "If set to true, use the project's TypeScript compiler version for API Extractor's analysis. API Extractor's included TypeScript compiler can generally correctly analyze typings generated by older compilers, and referencing the project's compiler can cause issues. If issues are encountered with API Extractor's included compiler, set this option to true."
```

Comment:
Explain that this corresponds to API Extractor's `--typescript-compiler-folder` CLI parameter and `IExtractorInvokeOptions.typescriptCompilerFolder` API. This will dispel some mystery, since people can read the API Extractor docs for more detail.